frontend/ui/data/creoptions.lua

```diff
@@ -528,6 +519,39 @@ Note that your selected font size is not affected by this setting.]]),
                     value_step = 1,
                 },
             },
+            {
+                name = "font_base_weight",
+                name_text = _("Font Weight"),
+                toggle = { "-1", "-½", "0", "+½", "+1", "+1½", "+2" },
+                values = { -1, -0.5, 0, 0.5, 1, 1.5, 2 },
+                args = { -1, -0.5, 0, 0.5, 1, 1.5, 2 },
+                default_value = 0,
+                event = "SetFontBaseWeight",
+                more_options = true,
+                more_options_param = {
+                    value_min = -3,
+                    value_max = 5.5,
+                    value_step = 0.25,
+                    precision = "%+.2f",
+                    value_hold_step = 1,
+                },
+                help_text = _([[Set the font weight delta from "regular" to apply to all fonts.
```

Comment:
That's technically a a weight [class](https://docs.microsoft.com/en-us/typography/opentype/spec/os2#usweightclass), but since we're not actually displaying that to the user, I'm okay with the simplification ;).